src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -75,8 +75,8 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
         {linkItems.map((linkItem) => (
```

Comment:
ok so you should add these props to the Box in line 74 (here's a guide to flexbox if u wanna look at it https://css-tricks.com/snippets/css/a-guide-to-flexbox/):
```
        display="flex"
        flexDirection={'column'}
        flex={1}
        justifyContent={'space-between'}
```